colour/models/rgb/transfer_functions/bt_709.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+ITU-R BT.709-6 OETF (OECF) and EOTF (EOCF)
+==========================================
+
+Defines *ITU-R BT.709-6* OETF (OECF) and EOTF (EOCF):
+
+-   :func:`oetf_BT709`
+-   :func:`eotf_BT709`
+
+See Also
+--------
+`RGB Colourspaces IPython Notebook
+<http://nbviewer.ipython.org/github/colour-science/colour-ipython/\
+blob/master/notebooks/models/rgb.ipynb>`_
+
+References
+----------
+.. [1]  International Telecommunication Union. (2015). Parameter values for
+        the HDTV standards for production and international programme exchange
+        BT Series Broadcasting service. In Recommendation ITU-R BT.709-6
+        (Vol. 5, pp. 1–32). Retrieved from https://www.itu.int/dms_pubrec/\
+itu-r/rec/bt/R-REC-BT.709-6-201506-I!!PDF-E.pdf
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+
+from colour.utilities import as_numeric, warning
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013 - 2015 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Production'
+
+__all__ = ['oetf_BT709',
+           'eotf_BT709']
+
+
+def oetf_BT709(value):
+    """
+    Defines *Recommendation ITU-R BT.709-6* OETF (OECF) opto-electronic
+    transfer function.
```

Comment:
Should oetf be put after the full term to make the function name clearer? Same for following functions.